### DISCUSSION TITLE
How to add  other tools to neurodesktop

### DISCUSSION THREAD

**Discussion by Yibh22** [neurodesk/neurodesk.github.io-hugo#406] [Q&A]
Dear experts,

I am new to Neurodesktop. I successfully installed Neurodesktop on Ubuntu 22.04 using ternimals. And I want to add some other tools, such as micapipe, to it. How do I do that? 

Actually, I found a script at https://github.com/NeuroDesk/neurocontainers/blob/master/recipes/micapipe/build.sh. Do I just need to run the build.sh script in the terminal to add this tool to Neurodesktop? Moreover, If I try to download the newest version of micapipe, is it ok to just change the export toolVersion='v0.1.2' to export toolVersion='v0.2.3' ？

Thank you for your kind consideration, and I am looking forward to your reply.

**Comment by Yibh22** [comment]
Dear experts,

I also found a website for creating a new app: https://www.neurodesk.org/developers/new_tools/manual_build/#create-a-new-app. However, it is for developers and too complicated for me (new user) to understand. So I was wondering what the easiest way is to add the latest version of micapipe to Neurodesktop so that I can directly use this application in Neurodesktop.

I look forward to your reply!

**Comment by stebo85** [comment]
Dear @Yibh22,

To add a new application to Neurodesk it needs to be committed to the neurocontainers repository. We started with adding micapipe but it wasn't completed - it would be wonderful if you could help to add it.

Thank you
Steffen
